src/components/rhelGraphCard/rhelGraphCard.js

```diff
@@ -98,7 +128,9 @@ RhelGraphCard.propTypes = {
     usage: PropTypes.array
   }),
   pending: PropTypes.bool,
-  t: PropTypes.func
+  t: PropTypes.func,
+  breakpoints: PropTypes.object,
```

Comment:
future:
if it looks like the breakpoints implementation isn't going to disappear in the future, and we're dependent on specific properties coming across it would probably be to our benefit to convert this to a shape and call the props out

Reply:
can do